### PR TITLE
Made accessibility related changes.

### PR DESCRIPTION
**Pull request by tejasvikolhe01**

### CODE REVIEW

File: resources/js/Pages/Auth/Register.vue [hunk 1]
```diff
@@ -27,7 +27,7 @@ const submit = () => {
     <GuestLayout>
         <Head title="Register" />
         <div class="container">
-            <form @submit.prevent="submit" class="col-lg-5 offset-lg-3">
+            <form role="form" @submit.prevent="submit" class="col-lg-5 offset-lg-3" aria-label="Register User">
```
Comment (alex-bowers) [inline resources/js/Pages/Auth/Register.vue:30]: As you are using a form element, you don't need to define `role="form"`. That's only needed if you were wrapping form elements inside of a div. Does that make sense?

File: resources/js/Pages/Dashboard.vue [hunk 2]
```diff
@@ -63,20 +63,28 @@ import { ref } from 'vue';
...
                             </div>
                         </div><!-- post-item end -->
-                        <PrimaryButton v-if="this.posts.length>3 && this.postLength < this.posts.length" @click="loadMore" class="offset-lg-5 col-md-3">
-                            Load More
-                        </PrimaryButton>
+                        <!-- pagination -->
+                        <nav role="navigation" aria-label=”pagination”>
+                            <ul class="pagination float-end">
+                                <li v-for="(link,index) in loadMoreLinks"  class="page-item">
+                                    <a v-bind:class="{ active: link.active }" class="page-link" @click="loadMorePosts(link.url)" v-if="index>0 && index<loadMoreLinks.length-1" href="javascript:void(0)">{{ index }}</a>
```
Comment (alex-bowers) [inline resources/js/Pages/Dashboard.vue:84]: You don't need to type `v-bind`, `:class` will also work.

Comment (alex-bowers) [reply resources/js/Pages/Dashboard.vue:84]: Also, why have you used `javascript:void(0)`?

Comment (tejasvikolhe01) [reply resources/js/Pages/Dashboard.vue:84]: this will get removed as I have already resolved these things in my earlier PR.

**Review by alex-bowers — APPROVED** [review]

**Pull request merged**